corehq/apps/users/models.py

```diff
@@ -2548,7 +2550,3 @@ def get(self, user_id):
             user = CouchUser.get_by_user_id(user_id)
             self.cache[user_id] = user
             return user
-
-
-from .signals import *
```

Comment:
that's a good point, it looks like signals.py has some code it executes to setup post-save signals which now won't get setup until the import happens within the function calls above